### PR TITLE
Feature/hummingboard support

### PR DESCRIPTION
**Pull request by shaunmulligan**

### CODE REVIEW

File: pages/installing/gettingStarted-Humming.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+This is the application dashboard where all of the devices connected to your application will be shown along with their status and logs.
+
+Click the `Download Device OS` button to get the resin.io operating system image for your
+application. A dialog will appear prompting you to specify how your device connects to the internet, if you select wifi, make double sure to that your `SSID` and `passphrase` are correct for the wifi router you intend to connect to. The download can take a little while to get started, so be patient.
+
+While the file downloads, ensure your SD card is formatted in [FAT32][fat32] ([WikiHow][wikihow] has [instructions][wikihow_format] on how to do this).
+
+Once the download is finished you should have a `.img` file with a name like `resin-myApp-0.1.0-0.0.14.img` where myApp is the name you gave your application on the dashboard.
+
+Now we have to burn the downloaded image on to the SD card. There are a couple of ways to do this, depending on your host computer operating system. We have listed a few below.
```
Comment (Page-) [inline pages/installing/gettingStarted-Humming.md]: Can probably drop the `host computer` bit altogether

File: pages/installing/gettingStarted-Humming.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+Once downloaded, launch Pi Filler, and follow the on-screen prompts. The first thing it will ask is for you to locate your .img file. It mentions the Raspberry Pi, but you can ignore that, it doesn't make any difference.
+
+Locate the .img file in your Downloads folder. It should be named something like `resin-myApp-0.1.0-0.0.14.img`, now click "choose".
+
+You can now insert your microSD card into your host machine and click continue. PiFiller will look for your SD card and tell you when it finds it.
+
+__NOTE:__ make 100% sure that the SD card it finds is in fact the correct card.
+
+Click continue and piFiller will write the SD card. This can 5-25 minutes depending on your machine. Once this is done, skip down to [setting up your device](/#/pages/installing/gettingStarted-Humming.md#setting-up-your-device)
```
Comment (Page-) [inline pages/installing/gettingStarted-Humming.md]: `can 5-25` -> `can take 5-25`

File: pages/installing/gettingStarted-Humming.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
+# Getting Started With the Humming Board
+
+![Humming Board](/img/hummingBoard.png)
+
+## What You'll Need
+
+* A [Humming Board i2][humming-link] from [Solid Run][solid-run]. Currently we only support the `i2` device type.
+
+* A 4GB or larger SD card.
+  The [Humming Board i2][humming-link] uses a Micro SD card. The [speed class][speed_class] of
```
Comment (Page-) [inline pages/installing/gettingStarted-Humming.md]: You use `Micro SD` here, but in a bunch of other places you use `microSD`, it's probably worth being consistent

File: pages/installing/gettingStarted-Humming.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+## Running Code On Your Device
+
+![git pushing](/img/screenshots/git_pushing.png)
+
+<!-- TODO: switch example_app link to point to right app. -->
+A good little project to get you started is this [simple webserver][example_app_node] written in node.js. It will respond with `Hello, World!` on port `:80`.
+
+To clone it, run the following in a terminal:-
+```
+git clone git@github.com:resin-projects/simple-server-node.git
```
Comment (Page-) [inline pages/installing/gettingStarted-Humming.md]: I think it's better to use the https method if you're an anonymous user/do not have a public key setup?  At least it's the url github gives me when visiting a repository I don't have write access to

**Comment by craig-mulligan** [comment]
@shaunmulligan okay to merge?

**Pull request merged**